website/src/components/Event.vue

```diff
@@ -46,6 +46,9 @@ export default {
     endDate () {
       return this.formatDate(this.event.end)
     },
+    summary (){
+      return Markdown.markdown([this.event.summary]);
```

Comment:
This part is unused. I think you should not have to create any extra method. Instead, the line 18:

```
        {{event.summary}}
```

should look something like:

```
        <markdown :text="event.summary"></markdown>
```

For more information, look at the components docs: https://vuejs.org/v2/guide/components.html